libs/utils/utils.js

```diff
@@ -152,7 +158,9 @@ export const [setConfig, getConfig] = (() => {
       document.documentElement.setAttribute('lang', config.locale.ietf);
       try {
         const contentDir = getMetadata('content-direction');
-        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || (config.locale.ietf ? (new Intl.Locale(config.locale.ietf)?.textInfo?.direction) : null) || 'ltr');
+        const locietf = config.locale.ietf;
+        const ietfDir = locietf ? (new Intl.Locale(locietf)?.textInfo?.direction) : null;
+        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || ietfDir || 'ltr');
```

Comment:
This seems backwards. Why would we do a calculation for something we don't know we will use?

```js
        const dir = getMetadata('dir')
          || config.locale.dir
          || new Intl.Locale(config.locale.ietf).textInfo.direction;
        documentElement.setAttribute('dir', dir || 'ltr');
```
Does this work?

Reply:
@auniverseaway Directly using `new Intl.Locale(config.locale.ietf).textInfo.direction;` will throw an error if config.locale.ietf is not set for a locale.
Could you please confirm if I should change the lines to following? -
```
        let dir = getMetadata('content-direction') || config.locale.dir;
        if (!dir && config.locale.ietf) {
          dir = new Intl.Locale(config.locale.ietf)?.textInfo?.direction;
        }
        document.documentElement.setAttribute('dir', dir || 'ltr');
```

Reply:
@chrischrischris Directly using new Intl.Locale(config.locale.ietf)?.textInfo?.direction will throw an error if ietf property is missing for a locale in the config.. 
One of the Unit test checks that if ietf and dir is not set it should fallback to 'ltr'
Could you please confirm if following can be used? -
```
        let dir = getMetadata('dir') || config.locale.dir;
        if (!dir && config.locale.ietf) {
          dir = dir || new Intl.Locale(config.locale.ietf)?.textInfo?.direction;
        }
        document.documentElement.setAttribute('dir', dir || 'ltr');
```

Reply:
@chrischrischris i think it's ok if we always set the 'dir' explicitly because it might be risky to rely on the browser to select proper direction.
e.g. in the [use case of this ticket](https://main--milo--adobecom.hlx.page/mena_ar/drafts/mathuria/test?martech=off), the 'lang' attr of the HTML tag says 'ar'. but the actual page content is in English, not sure what will be prioritised by the browser, but we want to set it to 'LTR'.